cluster/gce/config-test.sh

```diff
@@ -69,6 +69,7 @@ ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 # Default Log level for all components in test clusters and variables to override it in specific components.
 TEST_CLUSTER_LOG_LEVEL="${TEST_CLUSTER_LOG_LEVEL:---v=4}"
 KUBELET_TEST_LOG_LEVEL="${KUBELET_TEST_LOG_LEVEL:-$TEST_CLUSTER_LOG_LEVEL}"
+DOCKER_TEST_LOG_LEVEL="${DOCKER_TEST_LOG_LEVEL:---log-level=info}"
```

Comment:
Why have it default to `info` instead of empty?

1. That's redundant with the change in `kubernetes-e2e.yaml`;
1. If the default drifts elsewhere, this default won't drift with it, which is probably bad.